### PR TITLE
13 server error

### PR DESCRIPTION
**Pull request by alaa-yasin**
This pull request is to merge the server error page in our project which will be displayed to the user when there is a server error.

### CODE REVIEW

File: src/containers/errors/ServerError/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const ServerError = () => {
+  return (
+    <div className="servererror">
+      <Image className="servererror__img" />
+      <p className="servererror__msg">
+        Oh sorry about this problem our development team is working to on it
+        now.
+      </p>
+      <Button className="servererror__btn">Go Home</Button>
```
Comment (AlaaTaima) [inline src/containers/errors/ServerError/index.js]: you import antd button but you didn't use it.
use antd for button ,
<Button type='primary'>Go Home </Button>

File: src/containers/errors/ServerError/style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: rgba(36, 36, 36, 0.8);
+}
+
+.servererror__btn {
```
Comment (AlaaTaima) [inline src/containers/errors/ServerError/style.css]: use style of antd button and you can edit it as you want

**Review by AlaaTaima — CHANGES_REQUESTED** [review]

**Review by fares98 — DISMISSED** [review]

File: src/routes/App/index.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import 'antd/dist/antd.css';
+import ServerError from '../../containers/errors/ServerError';
 import './style.css';
 
 const App = () => {
   return (
     <div className="App">
       <h1>Hello from App</h1>
+      <ServerError />
```
Comment (yosefanajjar) [inline src/routes/App/index.js]: Remove it from the app

File: src/containers/errors/ServerError/index.js (hunk 1 above)
Comment (alaa-yasin) [reply src/containers/errors/ServerError/index.js]: done

File: src/routes/App/index.js (hunk 3 above)
Comment (alaa-yasin) [reply src/routes/App/index.js]: Oh my bad =D

**Comment by Fatmasiam** [comment]
where is the PR description ?!
take a look here #65
and please put the PR on the project

File: src/containers/errors/ServerError/style.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  height: 40px;
+  left: 49px;
+  top: 391px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 22px;
+  color: #FFFFFF;
+}
```
Comment (Fatmasiam) [inline src/containers/errors/ServerError/style.css]: add new line

**Comment by Fatmasiam** [comment]
check the accessibility and the iPad view

**Comment by Fatmasiam** [comment]
fix the button color :smile: 
and make sure --> that the brabch up to date with the master :no_good_woman:

File: src/containers/errors/ServerError/style.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  width: 80%;
+  margin-top: 137px;
+}
+
+.server-error__msg {
+  width: 229px;
+  height: 66px;
+  margin: 30px auto 20px auto;
+  font-family: Open Sans;
+  font-style: normal;
```
Comment (AlaaTaima) [inline src/containers/errors/ServerError/style.css]: by default : 
   font-style : normal
   font-weight : normal;
 #### so you can omit it

File: src/containers/errors/ServerError/style.css [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+.server-error__img {
+  width: 80%;
+  margin-top: 137px;
+}
+
+.server-error__msg {
+  width: 229px;
+  height: 66px;
+  margin: 30px auto 20px auto;
+  font-family: Open Sans;
```
Comment (AlaaTaima) [inline src/containers/errors/ServerError/style.css:15]: If the font-family consist of two parts you should add ' '
```suggestion
  font-family: 'Open Sans';
```

File: src/containers/errors/ServerError/style.css (hunk 5 above)
Comment (AlaaTaima) [inline src/containers/errors/ServerError/style.css]: by default : 
   font-style : normal
   font-weight : normal;
 #### so you can omit it

Comment (alaa-yasin) [reply src/containers/errors/ServerError/style.css]: okay

File: src/containers/errors/ServerError/style.css (hunk 6 above)
Comment (alaa-yasin) [reply src/containers/errors/ServerError/style.css:15]: I found that font-family names consisting of space-separated names need not be quoted

**Review by fares98 — APPROVED** [review]

**Review by fares98 — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Review by AlaaTaima — APPROVED** [review]

**Review by yosefanajjar — APPROVED** [review]

**Pull request merged**